R/doeScreening.R

```diff
@@ -52,11 +52,22 @@ doeScreening <- function(jaspResults, dataset, options, ...){
                                      options[["PBruns"]],
                                      nrow(daewr::DefScreen(options[["numberOfFactorsScreen3"]],
                                                            options[["numberOfFactorsScreen2"]],
-                                                           options[["screeningCenterPoints"]]))),
+                                                           ifelse(options[["numberOfFactorsScreen2"]] == 0,
+                                                                  options[["screeningCenterPoints"]],
+                                                                  0)
+                                                           )
+                                          )
+                                     ),
                        resolution = ifelse(options[["screeningType"]] == "PBdes",
                                            "III",
                                            "IV"),
-                       centers = options[["screeningCenterPoints"]])
+                       centers = ifelse(options[["screeningType"]] == "PBdes",
+                                        options[["screeningCenterPoints"]],
+                                        ifelse(options[["numberOfFactorsScreen2"]] == 0,
+                                               options[["screeningCenterPoints"]],
+                                               0)
+                                        )
```

Comment:
please don't use `ifelse` where you mean `if (condition) ... else ...`
```suggestion
                       centers = if(options[["screeningType"]] == "PBdes") { options[["screeningCenterPoints"]]
                                 } else if (options[["numberOfFactorsScreen2"]] == 0) { options[["screeningCenterPoints"]]
                                 } else { 0 }

```